cmake/Buildforcetorque-yarp-devices.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2017  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Silvio Traversaro <silvio.traversaro@iit.it>
```

Comment:
I am  definitely not the author of this file. Author line are probably not really helpful, so we can just remove them. 
```suggestion
```